app/upgrades/v21/constants.go

```diff
@@ -13,7 +20,17 @@ var Upgrade = upgrades.Upgrade{
 	UpgradeName:          UpgradeName,
 	CreateUpgradeHandler: CreateUpgradeHandler,
 	StoreUpgrades: store.StoreUpgrades{
-		Added:   []string{},
+		Added: []string{
+			// new modules
+			auctiontypes.ModuleName,
+
+			// v47 modules
+			crisistypes.ModuleName,
+			consensustypes.ModuleName,
+			feegranttypes.ModuleName,
+			group.ModuleName,
+			nft.ModuleName,
```

Comment:
Lets remove these

Aren't we missing tendermint?

Reply:
What store upgrade do we need to do for comet?

Reply:
Removed the three other modules https://github.com/osmosis-labs/osmosis/pull/6413/commits/9cd9cdba26e611bfe866e329404de902fa8d357d